src/main/java/it/smartcommunitylab/aac/webauthn/persistence/WebAuthnYubicoCredentialsRepository.java

```diff
@@ -64,7 +66,7 @@ public WebAuthnYubicoCredentialsRepository(String provider,
     public Optional<ByteArray> getUserHandleForUsername(String username) {
```

Comment:
why the Optional return? If a missing user handle is an error please throw an exception, otherwise just return null.

Reply:
The class implements the [CredentialRepository interface](https://github.com/Yubico/java-webauthn-server/blob/fe7a55710d1219ea03c514de872d95de0a15eb45/webauthn-server-core/src/main/java/com/yubico/webauthn/CredentialRepository.java#L55) defined by Yubico's library. In this case, we are bound to return an Optional.